modules/tracking/samples/lasot_benchmark.py

```diff
@@ -0,0 +1,240 @@
+import numpy as np
+import cv2 as cv
+import argparse
+import os
+
+
+def calc_coords(bb):
+    '''
+    The method used to evaluate coordinates of corners of the bounding box.
+    Input: tuple (x, y, w, h), where:
+    x - "x" coordinates of the top left corner
+    y - "y" coordinates of the top left corner
+    w - width of the bounding box
+    h - the height of the bounding box
+    Output: dictionary coordinates contain names of the values as keys and
+    numerical values as values
+    '''
+    xmin = bb[0]
+    xmax = bb[0] + bb[2] - 1.0
+    ymin = bb[1]
+    ymax = bb[1] + bb[3] - 1.0
+    cx = bb[0] + (bb[2] + 1.0) / 2
+    cy = bb[1] + (bb[3] + 1.0) / 2
+    coords = {'xmin': xmin, 'xmax': xmax, 'ymin': ymin, 'ymax': ymax,
+              'cx': cx, 'cy': cy}
+    return coords
+
+
+def get_iou(new, gt):
+    '''
+    During the calculation of intersection over union, we are checking
+    numerical value of area_of_overlap, because if it is equal to 0,
+    we have no intersection.
+    Lists, generated by 'calc_coords', are used as input for these methods.
+    Respectively:
+    new[0],gt[0] -> xmin, new[1],gt[1] -> xmax, new[2],gt[2] -> ymin
+    new[3],gt[3] -> ymax, new[4],gt[4] -> cx, new[5],gt[5] -> cy
```

Comment:
I think this description is redundant

Reply:
I thought that it improves understandability of the code - it should help to new user understand all calculations.

Reply:
```
new_xmin, new_xmax, new_ymin, new_ymax, new_cx, new_cy = new
gt_xmin, gt_xmax, gt_ymin, gt_ymax, gt_cx, gt_cy = gt
```
On these lines it is better to understand all the calculations.

Reply:
OK.